tests/integration_tests/test_xgboost.py

```diff
@@ -62,15 +87,19 @@ def objective(trial: optuna.trial.Trial) -> float:
 
 
 def test_xgboost_pruning_callback_cv() -> None:
+    if use_callback_cls:
+        key = "test-logloss"
+    else:
+        key = "test-error"
```

Comment:
repeating "Hmm, I think we don't need this block because I guess the CI will install xgboost>=1.3 hereafter and this block will not be tested."